src/test/java/integration/ShipmentGroupControllerIT.java

```diff
@@ -188,13 +190,13 @@ public void updateShipmentGroup() throws Exception {
                 put("/shipment-groups/{uuid}", shipmentGroupUuid.toString()).
         then().
                 statusCode(SC_OK);
-        long timeFinished = currentTimeMillis();
+        LocalDateTime timeFinished = now();
 
         // check updated data
         ShipmentGroup updatedShipmentGroup = shipmentGroupService.getEntityById(shipmentGroupUuid, user);
-        long timeModified = updatedShipmentGroup.getLastModified().getTime();
+        LocalDateTime timeModified = updatedShipmentGroup.getLastModified();
 
-        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished >= timeModified && timeModified >= timeStarted);
+        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished.isAfter(timeModified) && timeModified.isAfter(timeStarted));
```

Comment:
Plz, change. isAfter is not enough. It should do ">=".
Plz, create helper method assertDateBetween(date, dateFrom, dateTo) in the new class AssertHelper(package util).